src/influence_diagram.jl

```diff
@@ -971,9 +976,10 @@ function generate_diagram!(diagram::InfluenceDiagram;
 
         diagram.U = DefaultPathUtility(V_I_j_indexed, get_values(diagram.Y))
         if positive_path_utility
-            diagram.translation = 1 -  minimum(diagram.U(s) for s in paths(get_values(diagram.S)))
+            #DOES CHANGING THE DIAGRAM.TRANSLATION CALCULATION LIKE THIS CAUSE ANY ISSUES IF THERE ARE E.G. FORBIDDEN PATHS OR PATHS THAT NEVER HAPPEN. PROBABLY DEPENDS ON HOW THIS IS USED.
```

Comment:
I don't think it does. If the translation is too big, that doesn't break things as it still makes the utilities all negative or positive